get_best_focus_example.py

```diff
@@ -0,0 +1,34 @@
+import cv2
+import numpy as np
+
+import argparse
+from process import DataWangler
+import glob
+import os
+
+def get_best_focus_laplacian(image_list):
+    # Convert the image to grayscale if it is not already
```

Comment:
This comment seems not valid. Specifically, the below code does not convert the image to grayscle.